src/components/static/pages/AboutUs.tsx

```diff
@@ -2,116 +2,257 @@ import * as React from "react";
+import JeremiahReal from "../../../assets/images/profile/jeremiah-profile-irl.png";
+import Oliver from "../../../assets/images/profile/oliver-thumbnail.png";
+import OliverReal from "../../../assets/images/profile/oliver-irl.png";
+import Roy from "../../../assets/images/profile/roy-thumbnail.png";
+import RoyReal from "../../../assets/images/profile/roy-profile-irl.png";
+import Savoy from "../../../assets/images/profile/savoy-thumbnail.png";
+import SavoyReal from "../../../assets/images/profile/savoy-irl.png";
+import Izzy from "../../../assets/images/profile/izzy-thumbnail.png";
+import IzzyReal from "../../../assets/images/profile/izzy-irl.png";
+import Aaron from "../../../assets/images/profile/aaron-thumbnail.png";
+import AaronReal from "../../../assets/images/profile/aaron-profile-irl.png";
+import Dee from "../../../assets/images/profile/dee-thumbnail.png";
+import DeeReal from "../../../assets/images/profile/dee-profile-irl.png";
 
-const AboutUsWrapper = styled.div`
-  background-color: #d2d2d2;
-  height: ${contentHeight}px;
-`;
+import headerimg from "../../../assets/images/our-team/team.png";
+import backgroundImageWithNuvi from "../../../assets/images/2018_0814_Pattern_Adjusted.svg";
 
-const QuoteWrapper = styled.div`
-  background-color: #d2d2d2;
-  height: 430px;
-`;
+import {
+    Col,
+    Grid,
+    Row
+} from "react-bootstrap";
 
-const VisionWrapper = styled.div`
-  background-color: #ffffff;
-  height: 370px;
-`;
 
-const VisualWrapper = styled.div`
-  background-color: #d2d2d2;
-  height: 370px;
+const Header = styled.div`
+    background-image: url(${headerimg});
+    width: 100%;
+    height: 100%;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position:center;
+    background-size:cover;
+    box-shadow:0 1px 0 black,0 2px 0 rgba(255,255,255,0.15);
+    padding: 300px;
+    font-size: 100px;
+    text-align: center;
+    color: white;
+
 `;
-const MissionWrapper = styled.div`
-  background-color: #ffffff;
-  height: 370px;
+
+const HeaderText = styled.div`
+       
+    text-align: center;
+    color: white;
+    font-size: 100px;
+    opacity: 0;
+    animation: fade-in-right ease 2s forwards;
+`
+
+const Background = styled.div`
+  background-repeat: none;
+  background-image: url(${backgroundImageWithNuvi});
+  font-family: "Lato", sans-serif;
+  
 `;
-const MessageWrapper = styled.div`
-  background-color: #d2d2d2;
-  height: 370px;
+
+const Bio = styled.div`
+    font-family: "Lato", sans-serif;
+    width: 700px;
+    margin: 100px;
+    padding: 50px;
+    background: #d0e6e8;
+    font-size: 18px;
 `;
-const OurTeamWrapper = styled.div`
-  background-color: #ffffff;
-  height: 370px;
+
+const Bio1 = styled(Bio)`
+    background: #fce68f
+
 `;
-const BoardWrapper = styled.div`
-  background-color: #ffffff;
-  height: 370px;
+
+const BioPic = styled.div`
+    position: relative;
+    display: inline-block;
+    margin:100px;
+    float: center;
```

Comment:
I'm getting a message on my editor for this line, maybe something to take a look at - 

`inline-block is ignored due to the float. If 'float' has a value other than 'none', the box is floated and 'display' is treated as 'block'ts-styled-plugin(9999)`

Also I think the value for this float property is invalid, it will take either `none`, `left`, or `right`.